### PR TITLE
expose CmdLine in apps function

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Requested [on Discord](https://discord.com/channels/847502280503590932/1123360878285312101/1123360878285312101).

Disabled by default. Needs the `with-function-cmdline` command option.

##### Test Plan

- Add `with-function-cmdline` to apps's command options.
- Check the "Functions" tab. This requires changing "CmdLine" to any other value because there is a hard-coded rule to hide this column from the "Edit columns" menu (will be fixed tomorrow).

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Review by thiagoftsm — APPROVED** [review]
PR is working as expected, LGTM!

File: collectors/apps.plugin/apps_plugin.c [hunk 1]
```diff
@@ -4538,10 +4548,10 @@ static void apps_plugin_function_processes(const char *transaction, char *functi
         // cmd
         buffer_json_add_array_item_string(wb, p->comm);
 
-#ifdef NETDATA_DEV_MODE
         // cmdline
-        buffer_json_add_array_item_string(wb, (p->cmdline && *p->cmdline) ? p->cmdline : p->comm);
-#endif
+        if (enable_function_cmdline) {
+            buffer_json_add_array_item_string(wb, (p->cmdline && *p->cmdline) ? p->cmdline : p->comm);
+        }
```
Comment (MrZammler) [inline collectors/apps.plugin/apps_plugin.c:4554]: Ilya, a question, if there is not cmdline, then we'll print the `p->comm` again, we want this?

**Review by MrZammler — APPROVED** [review]

File: collectors/apps.plugin/apps_plugin.c (hunk 1 above)
Comment (ilyam8) [reply collectors/apps.plugin/apps_plugin.c:4554]: I believe cmdline is always present unless the process is a zombie.

**Comment by andrewm4894** [comment]
@ilyam8 this is very cool - i'm happy to update docs somewhere to make this more discoverable. 

where would be best place? in [here](https://github.com/netdata/netdata/blob/master/collectors/apps.plugin/README.md)?

**Pull request merged**